src/vsg/CMakeLists.txt

```diff
@@ -426,6 +426,14 @@ target_link_libraries(vsg ${LIBRARIES})
 
 if (BUILD_SHARED_LIBS)
     target_compile_definitions(vsg INTERFACE VSG_SHARED_LIBRARY)
+else()
+    if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+        if (VSG_USE_DYNAMIC_CAST)
```

Comment:
Where in CMake scripts is VSG_USE_DYNAMIC_CAST set & controlled?  Is there a missing CMake option for this?